projects/fastify-capture/src/index.ts

```diff
@@ -0,0 +1,195 @@
+import type { Log, Entry, Header, Cookie } from 'har-format';
+import { FastifyReply, FastifyRequest } from 'fastify';
+import setCookieParser from 'set-cookie-parser';
+import cookieParser from 'cookie';
+import fs from 'fs';
+import path from 'path';
+import http from 'http';
+
+type FastifyCaptureOptions = {
+  harOutputDir?: string;
+};
+
+export const fastifyCapture = (options: FastifyCaptureOptions = {}) => {
+  const entries: Entry[] = [];
+
+  const harOutputDir = options.harOutputDir
```

Comment:
Should we default to the optic capture tmp dir? I think we could call this something like "optic-har-capture"+project-path for the capture spec hash

Reply:
[This](https://github.com/opticdev/optic/blob/12827ccacf54e497b0bf5512a5be4ab5ed562fc3/projects/optic/src/commands/oas/captures/capture-storage.ts#L4)? 

Looks a bit fancy for the job I don't know. With a local folder people can find the output easily I think.